firmware/tunerstudio/rusefi.input

```diff
@@ -2109,16 +2104,12 @@ cmd_set_engine_type_default					= "w\x00\x31\x00\x00"
 		field = "D6 pin", 								HD44780_db6
 		field = "D7 pin", 								HD44780_db7
 		
-	dialog = connectRight, "", 	yAxis
+	dialog = connection, "", 	yAxis
```

Comment:
can we rename to connectionDialog or anything in the next iteration? just to make full text search happier